day8/README.md

```diff
@@ -287,7 +288,6 @@ int main() {
     getline(cin, first);
     getline(cin, second);
     cout << "The Levenshtein distance between \"" << first << "\" and \"" << second << "\" is: " << dist(first, second) << endl;
-}
```

Comment:
Why did you remove this curly bracket?

Reply:
i was removing the conflict markers and erased the curly bracket by mistake.. apologies